blue_green_deploy.go

```diff
@@ -48,8 +55,39 @@ func (p *BlueGreenDeploy) DeleteAllAppsExceptLiveApp(appName string) {
 
 }
 
-func (p *BlueGreenDeploy) PushNewApp(appName string, route plugin_models.GetApp_RouteSummary, manifestPath string) {
+func (p *BlueGreenDeploy) GetScaleParameters(appName string) AppScaleParameters {
+	appModel, err := p.Connection.GetApp(appName)
+	if err != nil {
+		return AppScaleParameters{}
```

Comment:
Errors should be handled as a general rule. Not providing scale parameters will just mean default are used instead for this case, but we will add an explicit p.ErrorFunc("Could not push new version", err) call here instead then.